corehq/apps/callcenter/indicator_sets.py

```diff
@@ -78,7 +78,7 @@ def __init__(self, domain_name, domain_timezone, cc_case_type, user,
         self.override_cases = override_cases
         self.override_cache = override_cache
 
-        self.config = indicator_config or CallCenterIndicatorConfig.default_config()
+        self.config = indicator_config or CallCenterIndicatorConfig.default_config(domain_name)
```

Comment:
Pass in the domain name so that we can add in any custom indicators for the domain.